src/workers/stakers.ts

```diff
@@ -58,6 +58,7 @@ const processEraForExposure = (data: AnyJson) => {
         total,
         share,
         isValidator,
+        exposedPage: 0,
```

Comment:
On Polkadot:
(but they probably have a bug because they start from page 1)
https://github.com/polkadot-cloud/polkadot-staking-dashboard/blob/main/packages/app/src/workers/stakers.ts#L70-L71